repondeur/db_migrations/versions/a3962c18da56_new_models.py

```diff
@@ -72,7 +72,7 @@ def upgrade():
         sa.ForeignKeyConstraint(["lecture_pk"], ["lectures.pk"]),
         sa.ForeignKeyConstraint(["parent_pk"], ["amendements.pk"]),
         sa.PrimaryKeyConstraint("pk"),
-        sa.UniqueConstraint("num", "lecture_pk"),
+        sa.UniqueConstraint("num", "rectif", "lecture_pk"),
```

Comment:
I don't think we should include `rectif` in the UniqueConstraint, as we should not have e.g. nº42 and nº42 rectif. at the same time (it's a new version of the same amendement).